composer.json

```diff
@@ -87,7 +87,7 @@
         "sylius/resource": "^1.10",
         "sylius/resource-bundle": "^1.10",
         "sylius/theme-bundle": "^2.3",
-        "sylius/twig-hooks": "^0.1",
```

Comment:
I'm leaving it as `dev-main` in case of any bugs you might found. I'll release and bump to `0.2` once I decide to introduce new features and refactors.